lib/hw_unique_key/pm.yml.huk

```diff
@@ -1,8 +1,10 @@
 #include <autoconf.h>
 
+#ifdef CONFIG_SOC_NRF52840
```

Comment:
Would it be better if the size of this partition was set to 0 in kconfig if this condition was true?

From my point of view this information is more accesible to the user. I can see the point that the `this partition has been removed since its size is 0` on every build is a bit verbose, but you could consider it anyway.